### ISSUE TITLE
Exception when running kong db:migrations

### ISSUE THREAD

**Issue by thefosk** [Mashape/kong#104]
```
[root@7b0f43dd5667 /]# kong db migrations
Using config: /etc/kong/kong.yml
/usr/bin/lua: /usr/local/share/lua/5.1/kong/cli/db.lua:45: bad argument #1 to 'concat' (table expected, got nil)
stack traceback:
	[C]: in function 'concat'
	/usr/local/share/lua/5.1/kong/cli/db.lua:45: in main chunk
	[C]: in function 'require'
	/usr/local/lib/luarocks/rocks/kong/0.1.0beta-3/bin/kong:38: in main chunk
	[C]: ?
```

**Comment by thibaultCha** [comment]
Fixed by #105 ?

**Comment by thefosk** [comment]
Nope, this is a separate issue.
